src/server.cpp

```diff
@@ -1171,10 +1170,10 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
             }
         }
 
-        // convert from double to short with clipping
+        // clip the float data
         for ( i = 0; i < ( 2 * iServerFrameSizeSamples ); i++ )
         {
-            vecsSendData[i] = Double2Short ( vecdIntermProcBuf[i] );
+	    vecsSendData[i] = clipFloat( vecdIntermProcBuf[i] );
```

Comment:
something went wrong with the formatting.

Reply:
add a space before the "("

Reply:
Fixed.